src/pixie/images.nim

```diff
@@ -404,12 +404,12 @@ proc drawCorrect*(a: Image, b: Image, mat: Mat3, blendMode: BlendMode) =
 
 const h = 0.5.float32
 
-proc drawUberStatic(
-  a, b, c: Image,
-  start, stepX, stepY: Vec2,
+proc drawUber(
+  a, b: Image,
+  start, dx, dy: Vec2,
   lines: array[0..3, Segment],
   blendMode: static[BlendMode],
-  smooth: static[bool],
+  smooth: static[bool] = false
```

Comment:
I don't think this method should have a default, if some one is using drawUber they should know what they are doing.

The much nicer draw method should.

Reply:
I don't want to mistakenly call uber draw without thinking about smoothing.